config/application.rb

```diff
@@ -84,5 +84,34 @@ class Application < Rails::Application
     config.x.db_url = Rails.env.production? ? "postgres://#{ENV['POSTGRES_USER']}:#{ENV['POSTGRES_PASSWORD']}@#{ENV['POSTGRES_HOST']}:5432/#{ENV['POSTGRES_DATABASE']}" : 'postgres://localhost:5432/apply_for_legal_aid_dev' # rubocop:disable Layout/LineLength
 
     config.active_job.queue_adapter = :sidekiq
+
+    # list geckoboard updater jobs which are suspended by host environment
+    config.x.suspended_geckoboard_updater_jobs = {
+      development: %w[
+        Dashboard::FeedbackItemJob
+        Dashboard::ApplicantEmailJob
+        Dashboard::ProviderDataJob
+        Dashboard::UpdaterJob
+      ],
+      test: %w[
+        Dashboard::FeedbackItemJob
+        Dashboard::ApplicantEmailJob
+        Dashboard::ProviderDataJob
+        Dashboard::UpdaterJob
+      ],
+      uat: %w[
+        Dashboard::FeedbackItemJob
+        Dashboard::ApplicantEmailJob
+        Dashboard::ProviderDataJob
+        Dashboard::UpdaterJob
+      ],
+      staging: %w[
+        Dashboard::FeedbackItemJob
+        Dashboard::ApplicantEmailJob
+        Dashboard::ProviderDataJob
+        Dashboard::UpdaterJob
```

Comment:
We might want to leave the staging jobs enabled - I know Alice sometimes tests changes there before deploying to production. With UAT it should be fairly easy to enable/disable them per environment by pushing a commit to a PR, but with staging we would have to merge and go through nearly the whole pipeline which feels time consuming/risky.

Reply:
Agreed